install/resources/mybb_theme.xml

```diff
@@ -8455,6 +8455,14 @@ if(use_xmlhttprequest == "1")
 <td class="trow1"><strong>{$lang->new_forum}</strong></td>
 <td class="trow2">{$forumselect}</td>
 </tr>
+<tr>
```

Comment:
The parent `<template>` tag (`moderation_inline_movethreads`) should have its `version` attribute updated to `1822`.

Reply:
Requested changes have been made

Reply:
Brilliant, thanks!